vendor/mooltipass/mcCombinations.js

```diff
@@ -1551,7 +1555,8 @@ mcCombinations.prototype.retrieveCredentialsCallback = function (credentials) {
         }
         // Unsure about this restriction. Probably should always make a retrieve credentials call (need to think about it)
         else if (currentForm.combination) {
-            if (credentials[0].Login && currentForm.combination.fields.username) {
+			//if (credentials[0].Login && currentForm.combination.fields.username) {
```

Comment:
why did you comment out the previous? just make the change, that's what version control is for.